hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -25,6 +25,8 @@
 java_import org.apache.hadoop.hbase.util.Bytes
 java_import org.apache.hadoop.hbase.ServerName
 java_import org.apache.hadoop.hbase.TableName
+java_import org.apache.hadoop.hbase.client.ColumnFamilyDescriptorBuilder
```

Comment:
Are there imports you can drop now after doing this work? e.g. the HTableDescriptor import?

Reply:
As of my most recent commit, the symbols HTableDescriptor and HColumnDescriptor are not present in the hbase-shell module, which I verified via `grep -r 'HColumnDescriptor\|HTableDescriptor' hbase-shell`